waku/v2/node/rpc_api/waku_api_tools.nim

```diff
@@ -0,0 +1,21 @@
+import
+  waku_api_types,
+  options,
+  ../wakunode2
+
+## Conversion tools
+## Since the Waku v2 JSON-RPC API has its own defined types,
+## we need to convert between these and the types for the Nim API
+proc convertToAPI*(response: HistoryResponse): HistoryResponseAPI =
+  if response.pagingInfo != PagingInfo():
+    # PagingInfo is not empty. Set pagingInfo Option on API response.
+    result = HistoryResponseAPI(messages: response.messages, pagingInfo: some(response.pagingInfo))
```

Comment:
we avoid `result` as it turns off useful compiler warnings and is prone to bugs - `expression` return is preferred, followed by explicit `return` (expressions don't work in all contexts, such as `async`) - in this case, just remove `result =`

Reply:
Thanks. Addressed in 6bea1b3. General issue in #287.